plugins/Marketplace/resources/qml/Packages.qml

```diff
@@ -11,6 +11,12 @@ ListView
     id: packages
 
     property string pageTitle
+    property bool bannerVisible
+    property string bannerIcon
+    property string bannerBody
```

Comment:
It's not quite the body right? It contains the text of the banner. As such i think it's better to call it "bannerText"

Reply:
https://github.com/Ultimaker/Cura/pull/10948/commits/00135e574dc464f1562c277ea389511dbc6735fb